src/game.js

```diff
@@ -602,12 +626,11 @@ export default class Game {
 				turnTimePool: v.numeric_properties.turnTimePool,
 				unitDrops: v.numeric_properties.unitDrops,
 			};
-			if (v.string_properties.match_id) {
```

Comment:
![CleanShot 2021-12-02 at 17 10 23@2x](https://user-images.githubusercontent.com/199204/144375146-4f29d563-0e13-442a-8fed-120fcd8ece8e.png)

Here is an example of the bug I mentioned in the MR.